src/components/user/unavailability/unavailability.repository.ts

```diff
@@ -1,7 +1,11 @@
 import { Injectable } from '@nestjs/common';
 import { node, relation } from 'cypher-query-builder';
-import { ChangesOf } from '~/core/database/changes';
-import { ID, ServerException, Session } from '../../../common';
+import {
+  ID,
+  NotFoundException,
+  ServerException,
+  Session,
+} from '../../../common';
```

Comment:
```suggestion
} from '~/common';
```

Same for the one below while you're in there.